actions/role.js

```diff
@@ -1,105 +1,79 @@
-const userConfig = require('../../config')
-
-module.exports = role;
+module.exports = role
+const log = require('log-cb');
+
+role.command = 'role';
+role.usage = 'role <method> [name]'
```

Comment:
We have to have a consistent method through each command, so a help command can display the information.

Would we have a `$command <action> [...args]` for every command?  And if so, we should probably make a separate library file to simplify it. 

Also, can't set `.name` on functions.  It is a read-only property that comes from `function foo () {}` the reference name.